src/java.base/macosx/classes/apple/security/AppleProvider.java

```diff
@@ -28,7 +28,7 @@
 import java.security.*;
 import static sun.security.util.SecurityConstants.PROVIDER_VER;
 
-/**
+/*
  * The Apple Security Provider.
  */
```

Comment:
Does it make sense to incorporate this comment into the following javadoc comment?

```java
/**
 * Defines the (an) Apple security provider.
 * ...
 */
@SuppressWarnings("serial") // JDK implementation class
```

Reply:
Maybe, but only maybe, and if so, it would be out of scope for this work. That would be up to the relevant component team.

That being said, neither comment is part of any public API, and this comment is effectively already present in the first sentence of the actual doc comment.